flight_controller_2.py

```diff
@@ -17,17 +17,20 @@ def main():
     '''
     #------------------------------------------------------------------
 
-    #Here, the launch function awaits input from the user.
```

Comment:
I moved these inside the "try:" because the "finally:" would be skipped if I killed the program before sending the start signal.